openstates/scrape/base.py

```diff
@@ -138,8 +145,33 @@ def save_object(self, obj):
         self.output_names[obj._type].add(filename)
 
         if self.scrape_output_handler is None:
-            with open(os.path.join(self.datadir, filename), "w") as f:
-                json.dump(obj.as_dict(), f, cls=utils.JSONEncoderPlus)
+
+            file_path = os.path.join(self.datadir, filename)
+
+            # Remove redundant prefix
+            try:
+                file_path_ = file_path[file_path.index("_data") + 6 :]
```

Comment:
Magic number here (`6`). Can you give an example of _why_ you're trimming a portion off the beginning of the string?